packages/modelviewer.dev/examples/loading/index.html

```diff
@@ -465,6 +465,104 @@ <h4></h4>
       </div>
     </div>
 
+    <div class="sample">
+      <div id="useA11y" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <h4 id="intro">
+            <span class="font-medium">Use a11y.</span>
+            Improve accessibility, by adding a11y attribute to the
+            model-viewer element. To have more descriptive translations to
+            improve screen reader capability.
+          </h4>
+          <div class="heading">
+            <h2 class="demo-title">A11Y defined translations</h2>
+            <h4></h4>
+          </div>
+          <example-snippet stamp-to="useA11y" highlight-as="html">
+            <template>
+              <model-viewer
+                id="a11y-viewer"
+                loading="eager"
+                camera-controls
+                touch-action="pan-y"
+                src="../../shared-assets/models/Astronaut.glb"
+                shadow-intensity="1"
+                alt="A 3D model of a astronaut"
+                a11y='{"front": "The front of an 3D modeled astronaut with ports for different equipments and some controlls for the suit", "back": "The back of an 3D modeled astronaut showing its backpack with environmental system", "left": "The left side of an 3D modeled astronaut showing mission patch", "right": "The right side of an 3D modeled astronaut usally has the flag of the country of the astronaut"}'
+              >
+                <div class="a11y-container">
+                  <button class="a11y-btn selected" onclick="switchA11y(this, {
+                    front: 'The front of an 3D modeled astronaut with ports for different equipments and some controlls for the suit', 
+                    back: 'The back of an 3D modeled astronaut showing its backpack with environmental system', 
+                    left: 'The left side of an 3D modeled astronaut showing mission patch', 
+                    right: 'The right side of an 3D modeled astronaut usally has the flag of the country of the astronaut',
+                    'interaction-prompt': 'Use mouse, touch or arrow keys to move'
+                  }, 'A 3D model of a astronaut')">English <span class="a11y-icon">🇺🇸</span></button>
+                  <button class="a11y-btn" onclick="switchA11y(this, {
+                    front: 'Framsidan av den 3D-modellerade astronauten, med portar för olika utrustningar och några kontroller för dräkten', 
+                    back: 'Baksidan av den 3D-modellerad astronauten som visar sin ryggsäck med miljösystemet', 
+                    left: 'Den vänstra sidan av den 3D-modellerade astronauten, som visar uppdragslappen', 
+                    right: 'Den högra sidan av den 3D-modellerade astronauten, har vanligtvis flaggan för astronautens land',
```

Comment:
I think this will result in untranslated default strings for the upper and lower views, right? Can we go ahead and include those too?